Formula/google-java-format.rb

```diff
@@ -13,6 +13,7 @@ class GoogleJavaFormat < Formula
   end
 
   depends_on "maven" => :build
+  depends_on "openjdk" => :test
```

Comment:
I believe we need this in runtime as well

Reply:
I dont like adding java 14 as runtime. If I really need to have openjdk as runtime, I would rather have `openjdk@11`.

Reply:
Homebrew is an evergreen package manager. We use the latest openjdk if it works